From 112e3d237a27e3754afc88fee8c0d18e7682dd4f Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Wed, 9 Dec 2020 18:16:48 +0800
Subject: [PATCH] ANDROID: Incremental fs: Fix printk format warnings in
 format.c

fs/incfs/format.c: In function 'incfs_read_next_metadata_record':
./include/linux/kern_levels.h:5:18: warning: format '%ld' expects argument of type 'long int', but argument 2 has type 'size_t {aka unsigned int}' [-Wformat=]
fs/incfs/format.c:619:3: note: in expansion of macro 'pr_warn'
   pr_warn("incfs: The record is too large. Size: %ld",
   ^~~~~~~

Signed-off-by: Tao Huang <huangtao@rock-chips.com>
Change-Id: I1a0b1e32761dd4358ff3ca210041893373f0a1a5
---
 fs/incfs/format.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fs/incfs/format.c b/fs/incfs/format.c
index 52079b5a45e6..fb7417c4dee8 100644
--- a/fs/incfs/format.c
+++ b/fs/incfs/format.c
@@ -619,7 +619,7 @@ int incfs_read_next_metadata_record(struct backing_file_context *bfc,
 	md_record_size = le16_to_cpu(md_hdr->h_record_size);
 
 	if (md_record_size > max_md_size) {
-		pr_warn("incfs: The record is too large. Size: %ld",
+		pr_warn("incfs: The record is too large. Size: %zu",
 				md_record_size);
 		return -EBADMSG;
 	}
-- 
2.35.3

